Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -207,3 +215,9 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
 private func rounded(_ value: Double) -> Double {
   return round(value * 100) / 100
 }
+
+private func amountFromBacking(_ backing: Backing?) -> Double? {
+  guard let amount = backing?.amount else { return nil }
+
+  return amount - Double(backing?.shippingAmount ?? 0)
```

Comment:
I think it's usually better when doing arithmetic using money types to use `Decimal`:

```
let shippingAmount: Double = backing?.shippingAmount ?? 0
let pledgeAmount = Decimal(amount) - Decimal(shippingAmount)

return (pledgeAmount as NSDecimalNumber).doubleValue
```

What do you think?

Reply:
Hmm yeah good point - although we seem to have been a little inconsistent with this 🤦‍♂

https://github.com/kickstarter/ios-oss/blob/master/Library/ViewModels/BackingViewModel.swift#L176

But I'll change that here!